src/components/People/Human.tsx

```diff
@@ -0,0 +1,61 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const Human: React.FC<Props> = ({ person, people }) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug: personSlug,
+  } = person;
+  const { slug } = useParams();
+
+  const mother = people.find(p => p.name === motherName);
+  const father = people.find(p => p.name === fatherName);
+
```

Comment:
Would be better to prepare your array of people before rendering, and add `mother` and `father` values to every object. So you will no need to find it here on every render